honeybee_vtk/config.py

```diff
@@ -24,6 +160,17 @@ class DataConfig(BaseModel):
         ' data.'
     )
 
+    color_by: bool = Field(
```

Comment:
The name `color_by` gives me the feeling that you are looking for what value in several databases should be used to color it. Like temperature, humidity, velocity, ...

I also think this should be a single list outside each data config. 

'color_by': ['temperature', 'velocity']

Reply:
I am not clear what you mean here. But I captured this in an [issue](https://github.com/ladybug-tools/honeybee-vtk/issues/167) so that I talk with you about this on one of the calls.